src/app/accounts/PageAccounts.tsx

```diff
@@ -19,12 +19,10 @@ import { trpc } from '@/utils/trpc';
 export const PageAccounts = () => {
   const { t } = useTranslation('account');
 
-  const { data, isLoading: areAccountsLoading } = trpc.account.me.useQuery();
+  const { data: githubAccount, isLoading: isGithubLoading } = trpc.account.chooseProvider.useQuery({provider: 'github'});
+  const { data: gitlabAccount, isLoading: isGitlabLoading } = trpc.account.chooseProvider.useQuery({provider: 'gitlab'});
 
-  const githubAccount = data?.find((account) => account.provider === 'github');
-  const gitlabAccount = data?.find((account) => account.provider === 'gitlab');
-
-  const isLoading = areAccountsLoading;
+  const isLoading = isGithubLoading && isGitlabLoading 
```

Comment:
```suggestion
  const isLoading = isGithubLoading || isGitlabLoading 
```
at least one of them is loading , that means accounts are loading